test/unit/spec/common/cellComponents/tabs/jobActionDropdown-Spec.js

```diff
@@ -134,28 +134,53 @@ define([
         });
 
         describe('button state', () => {
-            let container;
+            let container, jobActionDropdownInstance;
+
+            /**
+             * Test the expected state of the job action dropdown buttons
+             *
+             * @param {array}   expectations in the form [
+             *                      ['querySelectorString', expectedValue],
+             *                      ['querySelectorString', expectedValue],
+             *                  ]
+             *                  where querySelectorString is used to select elements
+             *                  using container.querySelectorAll
+             *
+             *                  e.g. ['.dropdown [data-action]', true]
+             *                  expect all elements with a data-action attribute to be disabled
+             *
+             *                  if no expectations are supplied, it is assumed that
+             *                  all buttons are disabled
+             */
+            function expectButtonState(expectations) {
```

Comment:
One of these tests was failing, and I refactored them to DRY them out a bit when I fixed it.